cmd/medco-cli-client/main.go

```diff
@@ -108,10 +108,20 @@ func main() {
 			Name:  "startConcept, s",
 			Usage: "Survival start concept",
 		},
+		cli.StringFlag{
+			Name:  "startsWhen, w",
+			Usage: "Should it take the earliest or latest start event",
```

Comment:
Usage of should in "usage:" is weird, rather something like "it takes". Anyway it's not really clear this description, can you improve it? Also below.